docker/tpu/tpu_example.jsonnet

```diff
@@ -0,0 +1,25 @@
+local base = import 'templates/base.libsonnet';
```

Comment:
is this Drine.io config?

Reply:
We use `jsonnet` as the templating engine and the output is a Kubernetes config (.yaml)